src/steamtricks

```diff
@@ -10,10 +10,10 @@ STEAMTRICKS_SCRIPT="$(cd "${0%/*}" && echo "$PWD")/${0##*/}"
 STEAMTRICKS_DIR="$(dirname "$STEAMTRICKS_SCRIPT")"
 STEAMTRICKS_DATA_DIR=/usr/share/steamtricks
 STEAMTRICKS_DATA_REPO_DIR="$STEAMTRICKS_DATA_DIR/data"
-PID_FILE=~/.steamtrickspid
 CONFIG_DIR=~/.local/share/steamtricks
 CONFIG_RC="$CONFIG_DIR/steamtricksrc"
 CONFIG_CACHE_DIR="$CONFIG_DIR/cache"
+PID_FILE="$CONFIG_DIR/.steamtrickspid"
```

Comment:
steam uses a symlink from ~/ to .local on mine. 

pid is used after steamtricks_config which creates the folders. There is a pid check above that but works correctly if the folders and pid doesn't exist.